common/ethnicities/wc_ethnicities_highelf_rework.txt

```diff
@@ -0,0 +1,472 @@
+﻿@beauty1min = 0.35
+@beauty1max = 0.65
+
+@beauty2min = 0.4
+@beauty2max = 0.6
+
+@beauty3min = 0.45
+@beauty3max = 0.55
+
+# Warcraft
+###Skin colors
+@human_skin_color_x_min = 0.15
+@human_skin_color_x_max = 0.2
+
+###Hair colors
+@blonde_color_x_min = 0.2
+@blonde_color_x_max = 0.25
+@blonde_color_y_min = 0.15
+@blonde_color_y_max = 0.62
+
+@brown_color_x_min = 0.08
+@brown_color_x_max = 0.2
+@brown_color_y_min = 0.86
+@brown_color_y_max = 0.95
+
+@red_color_x_min = 0.05
+@red_color_x_max = @brown_color_x_min
+@red_color_y_min = 0.53
+@red_color_y_max = 0.65
+
+@white_color_y_min = 0.05
+@white_color_y_max = 0.05
+
+###Eye colors
+@gold_color_x_min = 0.25
+@gold_color_x_max = 0.31
+@gold_color_y_min = 0.34
+@gold_color_y_max = 0.62
+
+@blue_eye_color_x_min = 0.6
+@blue_eye_color_x_max = 0.7
+
+@arcane_color_y_min = 0.34
+@arcane_color_y_max = 0.53
+
+@green_color_x_min = 0.33
+
+@elven_red_color_x_max = 1
+
+@blue_color_x_min = @blue_eye_color_x_min
+@blue_color_x_max = 0.83
+
+highelf_rework = {
+	template = "nelf_rework"
+	
+	hair_color = {
+		## Blonde
+		#40 = { @blonde_color_x_min @blonde_color_y_min @blonde_color_x_max @blonde_color_y_max }
+		## Brown
+		#5 = { @brown_color_x_min @brown_color_y_min @brown_color_x_max @brown_color_y_max }
+		## Red
+		#20 = { @red_color_x_min @red_color_y_min @red_color_x_max @red_color_y_max }
+		## White
+		#30 = { @brown_color_x_min @white_color_y_min @brown_color_x_max @white_color_y_max }
+		40 = { 0.118 0.388 0.176 0.447 } #Blonde/Yellow
+		30 = { 0.71 0.078 0.71 0.239 } #White/Silver
+		20 = { 0.004 0.443 0.067 0.541 } #Red/Orange
+		5 = { 0.055 0.820 0.078 1 } #Brown/Black
+	}
+	skin_color = {
+		10 = { 0.055 0.192 0.078 0.250 }
+	}
+	eye_color = {
+        ## Blue
+        #75 = { @blue_eye_color_x_min @arcane_color_y_min @blue_eye_color_x_max @arcane_color_y_max }
+		## Purple
+        #25 = { 0.85 0.15 0.95 0.43 }
+		## Gold
+		#1 = { @gold_color_x_min @gold_color_y_min @gold_color_x_max @gold_color_y_max }
+		75 = { 0.659 0.300 0.729 0.408 } #Blue
+		25 = { 0.784 0.300 0.859 0.333 } #Purple
+		1 = { 0.102 0.373 0.184 0.569 } #Gold
+	}
+	
+	#Height
+	gene_height = {
+        1 = { name = normal_height  range = { 0.5 0.6 }      }
```

Comment:
They're about the same size as humans so 0.5 on average.

Reply:
> 
> 
> They're about the same size as humans so 0.5 on average.

Changed this in 1e5e464e32fa5e2b7a77346381fbf8c773871852. The new range is 0.495 - 0.525, so there's a little bit of variety, but on average they're ~0.5.